akka-stream/src/main/scala/akka/stream/SslTlsOptions.scala

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (C) 2015-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream
```

Comment:
check the new QuickStart guide:
~~~java
import akka.stream.*;
import akka.stream.javadsl.*;
~~~
This is how you start a project.